### DISCUSSION TITLE
CSL 样式使用 Gitee pages

### DISCUSSION THREAD

**Discussion by zepinglee** [zotero-chinese/wiki#74] [好点子 / Ideas]
最近有些用户反映“[从 GitHub 安装样式](https://github.com/zotero-cn/styles/raw/main/src/manual-of-legal-citation-multi-lingual/manual-of-legal-citation-multi-lingual.csl)”的链接不能点开，似乎是因为 <raw.githubusercontent.com> 在大部分地区没有梯子的情况下无法访问。所以目前 <https://zotero-cn.github.io/styles/> 的实用性还比较有限。

另外最近还更新了一些 CSL 样式（<https://github.com/redleafnew/Chinese-STD-GB-T-7714-related-csl>），需要在 redleafnew/Chinese-STD-GB-T-7714-related-csl 和 zotero-cn/styles 两个仓库都 commit 和 push 一遍，感觉比较繁琐。

希望能够优先考虑建立 Gitee pages，让“CSL 中文样式”的页面能被大家更容易访问。然后将 CSL 样式的源码集中在一处，以方便管理。

**Comment by windingwind** [comment]
据我了解gitee有建立github镜像仓库的功能，可将目前的仓库在gitee建立一个镜像，会自动同步
